### DISCUSSION TITLE
Disable Owner requirement from ESLZ ARM template

### DISCUSSION THREAD

**Discussion by Sajkrish** [Azure/Enterprise-Scale#919] [Q&A]
I am trying to deploy ESLZ Arm template in this link "https://github.com/Azure/Enterprise-Scale/blob/main/docs/reference/adventureworks/README.md" and it requires owner permission to do that as a pre requisite. Is it possible to remove the Global Admin and/or Owner requirement and run the template using contributor role

**Comment by jtracey93** [comment]
Hi @Sajkrish, unfortunately not as the high level permissions are required to make RBAC assignments etc. which contributor does not cover.

**Reply by Sajkrish** [reply]
Hi, While deploying the template, getting below error. Any idea how to get this fixed

Deployment template validation failed: 'The template variable 'deploymentUris' is not valid: The language expression property 'templateLink' doesn't exist, available properties are 'template, templateHash, parameters, mode, debugSetting, provisioningState'.. Please see https://aka.ms/arm-template-expressions for usage details.'. (Code: InvalidTemplate)

Regards,
Sajith
